server/src/entity/tag.ts

```diff
@@ -0,0 +1,33 @@
+import {
+    Entity,
+    PrimaryGeneratedColumn,
+    Column,
+    CreateDateColumn,
+    UpdateDateColumn,
+    DeleteDateColumn,
+    BaseEntity,
+} from "typeorm";
+@Entity()
```

Comment:
Need a blank between line 9 to line 10 ?

Reply:
Changes have been made